lib/Weathermap/Integrations/Cacti/WeatherMapCacti10ManagementPlugin.php

```diff
@@ -59,13 +59,13 @@ public function handleManagementMainScreen($request, $appObject)
         $locale = $this->manager->application->getLocale();
         print "<h3>This is the React UI below here</h3>";
         print "<h1>INCOMPLETE</h1>";
-        print '<style>@import "cacti-resources/mgmt/main.css";</style>';
+        print '<style>@import "/cacti/plugins/weathermap/cacti-resources/mgmt/main.css";</style>';
```

Comment:
It is more correct to do this:
above add `global $config;`
and replace the string with:
`        print '<style>@import "' . $config['url_path'] . 'plugins/weathermap/cacti-resources/mgmt/main.css";</style>';`

Reply:
This is the other one I was going to comment on last night, but didn't have the Cacti source handy to check the config item name. Thanks @pautiina.

Reply:
Should be done now.